src/components/comment.jsx

```diff
@@ -3,11 +3,13 @@ import Edit from './edit';
 import Delete from './delete';
 import { v4 as uuidv4 } from 'uuid';
 import Comments from '../pages/Comments';
+import Reply from './reply';
 
 const Comment = (props) => {
   const [replyClicked, setReplyClicked] = useState(false);
   const [replyValue, setReplyValue] = useState('');
   const [comment, setComment] = useState(props.comment);
```

Comment:
again does this need to be state?

Reply:
Did you mean line 6 or line 12 for this one?